index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
@maxrimue Thanks for clarifying that.

I'm a fan of the snowball effect here, but I'm afraid it might make specifying a version number other than the default (which could be a higher priority use-case) more cumbersome than necessary. I guess it's not too bad if we can translate a `null` description as "use the default description", e.g. `version('version', null, '4.0.0')`.

If we're concerned with the multi-parameter API, perhaps we should just accept an object?

```js
version({ option: 'version', version: '4.0.0', describe: 'Current version of this module' })
```

That would avoid the ambiguity of the parameters and keep everything on a level playing ground. It would also allow us to easily support other parameters, e.g. `alias`.

We could even support a "string or object" parameter, assuming we can make a decision on which use-case gets priority for the string (I would argue it should be the option flag, similar to the `.help()` method).

Don't want to throw a wrench into the gears of this PR, but what do you think?